startupscript/dataproc/startup.sh

```diff
@@ -935,6 +947,58 @@ fi
     ${RUN_PYTHON} ${HAIL_SCRIPT_PATH}
   fi
 
+  #################################
+  # Configure Proxy Agent Overrides
+  #################################
+
+  # Map the CLI server to the appropriate UI url
+  case $TERRA_SERVER in
+    verily-devel)
+      ui_base_url=terra-devel-ui-terra.api.verily.com
+      ;;
+    verily-autopush)
+      ui_base_url=terra-autopush-ui-terra.api.verily.com
+      ;;
+    verily-staging)
+      ui_base_url=terra-staging-ui-terra.api.verily.com
+      ;;
+    verily-preprod)
+      ui_base_url=terra-preprod-ui-terra.api.verily.com
+    ;;
+    verily)
+      ui_base_url=workbench.verily.com
+      ;;
+    *)
+      >&2 echo "ERROR: $TERRA_SERVER is not a known verily server."
+      exit 1
+      ;;
+  esac
+
+  # Insert a link element to render
```

Comment:
Updating CSS embedded in HTML here is pretty fragile.
I know that there are often no _great_ choices for this kind of customization.

However, for changes like these one, we should at least consider writing a little bit of python that could take advantage of libraries like https://docs.python.org/3/library/html.parser.html.

Short of reworking this, please add a comment like

```
# The banner.html file contains <style> wrapper tags and a series of CSS styles,
# and a block of embedded javascript (inside a <script> tag) that we want to update.
```